app/src/main/java/org/simple/clinic/patient/PatientConfig.kt

```diff
@@ -10,7 +12,8 @@ const val MAXIMUM_SQLITE_QUERY_LIMIT = 1000
 data class PatientConfig(
     val limitOfSearchResults: Int,
     val scanSimpleCardFeatureEnabled: Boolean,
-    val recentPatientLimit: Int
+    val recentPatientLimit: Int,
+    val dueDaysToDefaulterPatient: Period
```

Comment:
`dueDaysToDefaulterPatient` doesn't seem to be very descriptive.

Change this to `daysSinceLastVisitToConsiderDefaulter` instead? It's a bit longer, but a lot more clearer IMO.

Reply:
In addition, the type is a `Period` but the name of the property starts with `days`. If we change the instantiation of the period to be months instead, this name is then incorrect.

Change it `timeSinceLastVisitToConsiderDefaulter`?

Reply:
I want to specific about the smallest unit here.
Since I do not care about the `time` but about `days`, I would rather use `periodToDefaulterPatient` instead of `time`

Reply:
Renamed to `periodToDefaulterPatientSinceLastVisit`